base/deprecated.jl

```diff
@@ -505,6 +503,7 @@ function OverflowError()
 end
 
 # PR #22703
+import .LinAlg: Bidiagonal
```

Comment:
Had to add this (and similarly below) to be able to build the sysimg, otherwise I get
```
LoadError("sysimg.jl", 461, LoadError("deprecated.jl", 507, TypeError(:Union, "parameter", Type, Bidiagonal)))
```
I wonder why those are not needed to compile master?

Reply:
Curious :).